module/batchwhen/README.md

```diff
@@ -0,0 +1,27 @@
+# flyd-batchwhen
+
+Batches values from the second stream while the first stream is true. When
+the first stream is true, it will send a batch of all events from the first
+stream that have occured while the first stream was true. If an event from
+the second stream occurs while the first stream is false, it will be dispatched
+as a single element batch.
```

Comment:
This description is a bit hard to understand. Is the second sentence about when the first stream changes to `true`?